new/simple-manager-blueprint.yaml

```diff
@@ -36,52 +36,68 @@ inputs:
   #############################
   # Cloudify Modules
   #############################
+  rest_service_rpm_source_url:
+    type: string
+    default: http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/m5-RELEASE/cloudify-rest-service-3.3.0-m5_b275.x86_64.rpm
+
+  management_worker_rpm_source_url:
+    type: string
+    default: http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/m5-RELEASE/cloudify-management-worker-3.3.0-m5_b275.x86_64.rpm
+
+  amqpinflux_rpm_source_url:
+    type: string
+    default: http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/m5-RELEASE/cloudify-amqp-influx-3.3.0-m5_b275.x86_64.rpm
+
+  cloudify_resources_url:
+    type: string
+    default: https://github.com/cloudify-cosmo/cloudify-manager/archive/3.3m5.tar.gz
```

Comment:
the cloudify_resources_url, yes.
Ths above are rpm's specific to the version.. like any other package (agent, cli, etc..), so no.